### PR TITLE
Add SquadMemberScreen modal component

### PR DESCRIPTION
**Pull request by tfettig22**
## What does this PR do?
- [x] New feature (adds functionality)
- [] Debugging/Refactor
 
### Where should your reviewer start?
- SquadMemberScreen component, in the form of a modal on the MySquads component
### What (if any) features are you implementing?
- Adds modal to view a squad members' profile
### What (if anything) did you refactor?
- n/a
### What issues does this PR close?
- #18 
### Is there anything that you need from your teammates?
- Test out modal and make sure it works smoothly
### Any other comments, questions, or concerns?
- I am getting a warning about children with the same key being rendered.  It wont tell me which render method to check and I tried changing all the instances of 'key' in our project and I cant fix it.... would love some eyes on that. Also my Android emulator crashes every time (I think because of this warning) so Nick if you are able to test it on your phone that would be great!
### How Has This Been Tested?
- [x] I tested my changes in the browser
- [] I tested with Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by aspitz1 — APPROVED** [review]
Was there repeat codes in this and your other PR Tom? I pulled this down and it seems to work great, Thank you 👍

File: src/components/MySquads.js [hunk 1]
```diff
@@ -42,8 +44,30 @@ const MySquads = ({ userID }) => {
       .catch((error) => console.log(error));
   }, []);
 
+  const memberIconClickHandler = (id) => {
+    setSelectedUser({})
+    getSingleUser(id).then(data => {
+      setSelectedUser(data.data)
+      console.log(selectedUser)
```
Comment (NickLiggett) [inline src/components/MySquads.js:51]:
```suggestion
```

File: src/components/SquadMemberScreen.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  closeText: {
+    fontWeight: "bold"
+  },
+  rawg: {
+    height: "5%",
+    marginTop: 90,
+  },
+});
+
+export default SquadMemberScreen;
```
Comment (NickLiggett) [inline src/components/SquadMemberScreen.js:111]: Nice Component bro
B)

**Review by NickLiggett — APPROVED** [review]

**Comment by NickLiggett** [comment]
Nice Component bro
B)

**Pull request merged**